src/main/resources/db/migration/0250/V286__Applications.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE accelerator.application_statuses (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.applications (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    planting_site_id BIGINT NOT NULL REFERENCES tracking.planting_sites,
+    internal_name TEXT NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    status_id INTEGER NOT NULL REFERENCES accelerator.application_statuses,
```

Comment:
Do we need the status as a column, or should we just look for the latest in history?

Reply:
Having it in the table will make it easier to expose applications via the search API (no need for a view or a subquery). More efficient too, though I expect the frequency of use and the data volume will be small enough that that's not too big a factor in practice.

Reply:
Either way! I was just thinking about keeping data independent in theory but this is easier to query for sure!